corehq/apps/hqadmin/management/commands/stale_data_in_es.py

```diff
@@ -55,6 +58,8 @@ def get_csv_args(delimiter):
     }
 
 
+retry_on_es_timeout = retry_on(ESError, delays=[2**x for x in range(10)])
```

Comment:
In the worst case these retry delays will accumulate to >17 minutes, with the final wait being half of that (>8 minutes). Do you really want to wait that long before giving up? What about reducing it to just over 2 minutes total?
```suggestion
retry_on_es_timeout = retry_on(ESError, delays=[2**x for x in range(7)])
```

Reply:
Bump @sravfeyn